samples/nrf9160/modem_shell/src/srest/srest_shell.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SREST_RIP_SHELL_H
```

Comment:
Yeah, that (REST In Peace) was my original name for this command ;-)
Will fix this.

> A generic comment about naming, does the name have to include the word "simple"? It could just be "REST client" and the command "rest", IMHO.
I wanted to emphasize that this only supports only SIMPLE queries/responses. None kind of streaming and whole response need to fit in same buffer, just a request and a response...

Reply:
Ok, keeping "simple" is fine by me.

Reply:
I guess "simple" is fine although there is a chance this is developed to a more advanced version at a later stage.

Reply:
Having gone through the code, I'd probably think of renaming to just `rest` instead of `srest` and Simple REST to be more future proof.

Reply:
OK, let the `srest` rest in peace and let there be a command `rest`.